src/client/init.ts

```diff
@@ -0,0 +1,22 @@
+import App from '@/components/App.vue';
+import store from '@/store';
+import { NS_ENTITY } from '@/store/namespaces';
+import { ENTITY_INIT } from '@/store/entity/mutationTypes';
+import EntityInitializer from '@/common/EntityInitializer';
+import Vue from 'vue';
+
+export default (): Promise<Vue> => {
+	return new Promise<Vue>( ( resolve, reject ) => {
+		window.mw.hook( 'wikibase.entityPage.entityLoaded' ).add( ( entity: any ) => {
+			store.commit(
+				`${NS_ENTITY}/${ENTITY_INIT}`,
+				( new EntityInitializer() ).newFromSerialization( entity ),
```

Comment:
We should see for some error handling - for this step in particular - in the next iteration.